qemu/cfg/seabios_iommu.cfg

```diff
@@ -0,0 +1,39 @@
+# Please add "iommu=pt intel_iommu=on" to kernel line in host
+- seabios_iommu: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = seabios_iommu
+    only q35
+    no WinXP WinVista Win7 Win8 Win8.1 Win2003
+    no Win2008 Win2008..r2 Win2012 Win2012..r2
+    virtio_dev_disable_legacy = on
+    virtio_dev_disable_modern = off
+    extra_params = "-device intel-iommu,device-iotlb=on,intremap=on"
+    machine_type_extra_params = "kernel-irqchip=split"
+    virtio_dev_iommu_platform = on
+    virtio_dev_ats = on
+    images += " stg0 stg1"
+    image_name_stg0 = "images/stg0"
+    image_name_stg1 = "images/stg1"
+    image_size_stg0 = 200M
+    image_size_stg1 = 200M
+    data_image_size = 200M
+    force_create_image_stg0 = yes
+    force_create_image_stg1 = yes
+    remove_image_stg0 = yes
+    remove_image_stg1 = yes
+    blk_extra_params_stg0 = "serial=TARGET_DISK0"
+    blk_extra_params_stg1 = "serial=TARGET_DISK1"
+    drive_format_stg0 = virtio
+    drive_format_stg1 = scsi-hd
+    virtio_scsi:
+        drive_bus_stg1 += 1
+    nics = "nic0"
+    nic_model_nic0 = virtio
+    device_id_nic0 = nic0
+    Linux:
+        enable_guest_iommu = yes
```

Comment:
@ybduan 
There have two scenarios for iommu in the guest
1. iommu=pt intel_iommu=on
2. intel_iommu=on
how about the scenario2

Reply:
Updated, Thanks!